dts/arm/rpi_pico/rp2040.dtsi

```diff
@@ -184,6 +185,22 @@
 			status = "disabled";
 			#pwm-cells = <3>;
 		};
+
+		vreg: vreg@40064000 {
+			compatible = "raspberrypi,core-supply-regulator";
+			reg = <0x40064000 1>;
+			status = "okay";
+			raspberrypi,brown-out-detection;
+			raspberrypi,brown-out-threshold = <860000>;
+			vreg_core: CORE {
+				regulator-always-on;
+				regulator-boot-on;
+				regulator-initial-mode = <REGULATOR_RPI_PICO_MODE_NORMAL>;
+				regulator-allowed-modes = <REGULATOR_RPI_PICO_MODE_NORMAL>;
+				regulator-min-microvolt = <800000>;
+				regulator-max-microvolt = <1300000>;
```

Comment:
note: these flags are used to limit regulator operating range, not to describe hardware capabilities. So in general, they should be put at board level. Since this is put at the SoC DTS files, it looks like for example the always-on/boot-on is something that can't be configured, so driver doesn't need to implement enable/disable.

Reply:
I moved these properties into boards/.
And it left properties that need to keep the regulator from turning off only.